lms/djangoapps/instructor/tests/utils.py

```diff
@@ -55,8 +55,8 @@ class FakeTarget(object):
         """ Corresponding fake target for a fake email """
```

Comment:
Do you know why these were made inner classes?  Seems they'd be just as good at top level.